src/lwc/geFormFieldPicklist/geFormFieldPicklist.js

```diff
@@ -99,6 +99,31 @@ export default class GeFormFieldPicklist extends LightningElement {
         this._recordTypeId = id || this.defaultRecordTypeId;
     }
 
+    get picklistValues() {
+        if (this.isTrueFalsePicklist) {
+            return this.trueFalsePicklistValues();
+        }
+        return this._picklistValues;
+    }
+
+    set picklistValues(value) {
+        this._picklistValues = value;
+    }
+
+    trueFalsePicklistValues() {
```

Comment:
this function is defined twice, could we put in a module and import it everywhere we need it?